homework-1/index.html

```diff
@@ -0,0 +1,681 @@
+                <div class="slider__text-text">Malicious software that blocks access to a system until a ransom is paid.
+                </div>
+              </h3>
+            </div>
+            <div>
+              <h3 class="slider__title-wrapper">
+                <div class="slider__text-title">Software</div>
+                <div class="slider__text-text">Malicious software that blocks access to a system until a ransom is paid.
+                </div>
+              </h3>
+            </div>
+            <div>
+              <h3 class="slider__title-wrapper">
+                <div class="slider__text-title">SOC</div>
+                <div class="slider__text-text">Malicious software that blocks access to a system until a ransom is paid.
+                </div>
+              </h3>
+            </div>
+            <div>
+              <h3 class="slider__title-wrapper">
+                <div class="slider__text-title">Trojan Horse</div>
+                <div class="slider__text-text">Malicious software that blocks access to a system until a ransom is paid.
+                </div>
+              </h3>
+            </div>
+            <div>
+              <h3 class="slider__title-wrapper">
+                <div class="slider__text-title">VPN</div>
+                <div class="slider__text-text">Malicious software that blocks access to a system until a ransom is paid.
+                </div>
+              </h3>
+            </div>
+            <div>
+              <h3 class="slider__title-wrapper">
+                <div class="slider__text-title">Virus</div>
+                <div class="slider__text-text">Malicious software that blocks access to a system until a ransom is paid.
+                </div>
+              </h3>
+            </div>
+            <div>
+              <h3 class="slider__title-wrapper">
+                <div class="slider__text-title">Worm</div>
+                <div class="slider__text-text">Malicious software that blocks access to a system until a ransom is paid.
+                </div>
+              </h3>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    </div>
+  </section>
+  <!----------glossary-end----------->
+  <!----------FQAs-start----------->
+  <section class="fqas section" id="fqas">
+    <div class="container">
+      <div class="fqas__img border-img">
+        <img class="mobile_img" src="images/logo-fqas_mobile.png" alt="fqas">
+        <img class="dekstop_img" src="images/logo-fqas.png" alt="fqas">
+      </div>
+      <ul class="fqas-list block">
+        <li class="block__item">
+
+          <button type="button" class="fqas__link fqas__link-first">
+            <div class="tips-item-wrap">
+              <span class="fqas-item__num">Q</span>
+              <span class="fqas-item__title">What are your system requirements?</span>
+            </div>
+          </button>
+          <ul class="hidden block-text-next">
```

Comment:
Too many 'ul'  'li' please you don't need it herr

Reply:
fixed a bug

Reply:
done